app/frontend/styles/components/_tabs.scss

```diff
@@ -0,0 +1,64 @@
+.tabs-component {
+  margin-bottom: govuk-spacing(4);
+
+  @include govuk-media-query ($from: desktop) {
+    border-bottom: 1px solid $govuk-border-colour;
+    margin-bottom: govuk-spacing(7);
+  }
+
+  .govuk-link {
+    text-decoration: none;
+
+    &:visited {
+      color: $govuk-link-colour;
+    }
+
+    &:focus {
+      color: $govuk-focus-text-colour;
+    }
+  }
+
+  &-navigation {
+    &__list {
+      font-size: 0;
+      list-style: none;
+      padding: 0;
+    }
+
+    &__item {
+      display: inline-block;
+      font-size: 1rem;
+      line-height: 1.25;
+      margin-bottom: 0;
+      margin-right: govuk-spacing(4);
+      margin-top: 0;
+    }
+
+    &__link {
+      @include govuk-font(19);
+
+      display: block;
+      font-weight: bold;
+      padding-bottom: govuk-spacing(3);
+      padding-top: govuk-spacing(3);
+      position: relative;
+      text-decoration: none;
+
+      &:focus {
+        padding-bottom: govuk-spacing(2);
+      }
+
+    }
+  }
+
+  .tabs-component-navigation__link[aria-current]::before {
```

Comment:
I had to do this was due to stylelint issues. More details see here: 085ea38fee9104702fa5146ba79e372d054ff0d4
If you have another way of doing this without the stylelint issues, I am happy to fix!

Reply:
I don't think this is a problem